packages/blocks/src/paragraph-block/paragraph-block.ts

```diff
@@ -172,12 +175,25 @@ export class ParagraphBlockComponent extends NonShadowLitElement {
 
     const observer = () => {
       this._showTipsPlaceholder =
-        this.model.text.length === 0 && this.model.type === 'text';
+        this.model.text.length === 0 &&
+        this.model.type === 'text' &&
+        !this._isComposing;
     };
     this.model.text.yText.observe(observer);
     this._disposables = new DisposableGroup();
     this._disposables.add(this.model.propsUpdated.on(observer));
     this._disposables.add(() => this.model.text.yText.unobserve(observer));
+    // Workaround for virgo skips composition event
```

Comment:
This may be also required for list block, code block and other Virgo-based components. Any idea to avoid boilerplate?

Reply:
Another block does not have placeholder, so it is okay.

We can handle this scenario in `rich-text` when the list block needs a placeholder.